packages/fxa-content-server/app/scripts/lib/survey-filter.js

```diff
@@ -63,9 +63,13 @@ export const createConditionCheckFn = (valSource) => (comparator) => (key) => (
   // condition is not in the configs.
   const condVal = getConditionWithKey(conds, key);
   if (condVal === NONE) {
-    return true;
+    return { passing: true, value: undefined };
   }
-  return valSource(fetchFn)(comparator)(condVal);
+  const result = valSource(fetchFn)(comparator)(condVal);
+  return {
+    passing: result !== undefined,
```

Comment:
You need to replace `falseFn` with a function that returns `undefined` because of this change.